index.html

```diff
@@ -1,14 +1,179 @@
+            <img src="images/base/title.png" alt="Welcome to Conference" />
+            <div class="summary">
+                <p>A fun festival where people from more than 80 countries who believe in the value of sharing and
+                    openness and create positive change gather and share their thoughts will be held in Korea in
+                    October.</p>
+            </div>
+            <div class="location">
+                <h2>2015.10.14 (Wed) ~ 17 (Sat)</h2>
+                <p>@ National Museum of Korea, Art Center Nabi, etc.</p>
+            </div>
+        </div>
+    </section>
+    <section class="container program-wrap">
+        <div class="content clearfix">
+            <h3>Main Program</h3>
+            <div class="guide-bar"></div>
+            <div class="program-list clearfix">
+                <div class="program-card">
+                    <img src="images/base/program_icon_01.png" alt="conference" class="program-image" />
+                    <h4>Conference</h4>
+                    <p>Listen to lectures from speakers from around the world and learn about the latest trends in the
+                        world.</p>
+                </div>
+                <div class="program-card">
+                    <img src="images/base/program_icon_02.png" alt="conference" class="program-image" />
+                    <h4>CC exhibition</h4>
+                    <p>Let's meet the creations of artists from various fields who share the open spirit of CC.</p>
+                </div>
+                <div class="program-card">
+                    <img src="images/base/program_icon_03.png" alt="conference" class="program-image" />
+                    <h4>forum</h4>
+                    <p>We have time to share our thoughts and opinions with experts by topic.</p>
+                </div>
+                <div class="program-card">
+                    <img src="images/base/program_icon_04.png" alt="conference" class="program-image" />
+                    <h4>workshop</h4>
+                    <p>You can create your own creations using open source instead of just looking at them.</p>
+                </div>
+                <div class="program-card">
+                    <img src="images/base/program_icon_05.png" alt="conference" class="program-image" />
+                    <h4>CC party</h4>
+                    <p>Create opportunities to freely network with CC personnel from around the world..</p>
+                </div>
+            </div>
+            <a href="#" class="more-link">view full program </a>
+            <button class="join-btn" type="button">Join CC Global Summit 2015</button>
+        </div>
+
+    </section>
+
+    <section class="container speakers-wrap b-bottom clearfix">
+        <div class="content">
+            <h3> Featured Speakers</h3>
+            <div class="guide-bar"></div>
+            <div class="speakers-list " id="speakers-list">
+            </div>
```

Comment:
- [ ] Please make sure when we view the webpage on the phone screen, your webpage shows only two speakers and a More button. When we click on the More button it should display the remaining speakers.

See the image for reference

![image](https://user-images.githubusercontent.com/83803414/157530746-2e246778-5879-4bc9-9d73-e1eeac2c0813.png)